roles/multiqc/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 multiqc_repo: https://github.com/ewels/MultiQC.git
 multiqc_dest: "{{ sw_path }}/multiqc"
-multiqc_version: "v1.10"
+multiqc_version: "v1.10.1"
```

Comment:
We can update Multiqc on monthly. Also the latest version is 1.11 right now.

Reply:
thanks, updated it to the latest now

Reply:
reverted the multiqc update so that we can include it in the monthly PR instead.

Reply:
👍